lib/bike_club.rb

```diff
@@ -1,3 +1,17 @@
 class BikeClub
-
-end
\ No newline at end of file
+    attr_reader :name, :bikers
+  
+    def initialize(name)
+      @name = name
+      @bikers = []
+    end
+  
+    def add_biker(biker)
+      @bikers << biker
+    end
+  
+    def most_rides
+      @bikers.max_by { |biker| biker.rides.length }
```

Comment:
This will only count the number of key-value pairs, but we want the number of time elements in the value arrays. So  we want to do `biker.rides.values.flatten.length` to get those (like we did in the walkthrough today!)